api/types/backend/backend.go

```diff
@@ -160,7 +160,7 @@ type ImageInspectOpts struct {
 type CommitConfig struct {
 	Author              string
 	Comment             string
-	Config              *container.Config
+	Config              *container.Config // TODO(thaJeztah); change this to [dockerspec.DockerOCIImageConfig]
```

Comment:
Possibly this would help reduce the conversion juggling - I'd have to check though exactly where this field is all used 🤔